### DISCUSSION TITLE
new openmpi missed by autobump?

### DISCUSSION THREAD

**Discussion by rfiorella** [Homebrew/discussions#5617] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.23-22-g21bb3e1
ORIGIN: https://github.com/Homebrew/brew
HEAD: 21bb3e15bbc1916a209747b305f49e34925d419f
Last commit: 3 hours ago
Core tap HEAD: bd694395d34e831566f515483538c4f2723a567e
Core tap last commit: 14 minutes ago
Core tap JSON: 18 Sep 18:47 UTC
Core cask tap JSON: 18 Sep 18:47 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 10
HOMEBREW_SORBET_RUNTIME: set
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: 10-core 64-bit arm_blizzard_avalanche
Clang: 16.0.0 build 1600
Git: 2.46.1 => /opt/homebrew/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 15.0-arm64
CLT: 16.0.0.0.1.1724870825
Xcode: 16.0
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

A new version of Open MPI was released a while back but has not been picked up by autobump (I tried updating the package, but `brew bump-formula-pr` errors of course due to this being on the autobump list.

Current tapped version is a patched version of 5.0.3, but 5.0.5 is available. I couldn't find any other discussion of why this hasn't been upgraded yet - is there a reason for this or is this a bug in autobump?

Asking as I am trying to track down some reasons why code that used to compile on macOS-14.x-arm64 using openmpi, and compiles under macOS: 15.0-arm64 using mpich, does not compile using the Homebrew openmpi for macOS 15.0-arm64

**Comment by carlocab** [comment, marked as answer]
Not missed; it just takes time to fix failing tests: https://github.com/Homebrew/homebrew-core/pull/177756

**Reply by rfiorella** [reply]
Got it, thanks! Somehow missed it in the pull request list, must have just checked discussions and issues
